### PR TITLE
bump to v1.0.0-rc6

### PR DESCRIPTION
**Pull request by vbatts**
The vote for this version: https://groups.google.com/a/opencontainers.org/d/msg/dev/3wLazzsd4GI/2vi0gsxtAwAJ

Since the [rc5 release](https://github.com/opencontainers/runtime-spec/releases/tag/v1.0.0-rc5) there have 128 changesets merged:
#880
#881
#838
#877
#823
#876
#875
#868
#873
#840
#850
#809
#874
#871
#863
#852
#854
#735
#811
#827
#851
#847
#858
#860
#862
#870
#846
#855
#859
#865
#828
#849
#856
#837
#848
#801
#818
#816
#822
#807
#815
#833
#839
#820
#814
#836
#832
#821
#824
#826
#819
#803
#766
#724
#812
#805
#804
#798
#802
#700
#732
#681
#701
#702
#733
#736
#767
#790
#799
#800
#713
#796
#703
#793
#789
#730
#794
#737
#792
#756
#764
#781
#785
#786
#787
#784
#783
#788
#777
#775
#768
#769
#773
#778
#780
#770
#776
#774
#761
#765
#763
#750
#758
#757
#739
#748
#753
#751
#749
#752
#740
#744
#741
#706
#738
#721
#722
#723
#720
#630
#714
#712
#718
#710
#711
#709
#708
#717

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+	* config: Require strictly-postitive 'timeout' values (#764)
+	* config: Strengthen punt to kernel for valid capabilities strings
+	  (#766, #790)
+	* config: Forbid setting 'readonly' true on Windows (#819)
+	* config: Forbid setting mount 'type' entirely on Windows and forbid
+	  UNC paths and mapped drives in 'source' on Windows (#821)
+	* config-linux: Clearly require absolute path for namespace (#720)
+	* config-linux: RFC 2119 tightening for namespaces (#767)
+	* config-linux: Require at least one entry in
+	  linux.seccomp.sycalls[].names (#769)
```
Comment (estesp) [inline ChangeLog]: minor typo `linux.seccomp.syscalls[].names` (missing second **s** in syscalls)

Comment (vbatts) [inline ChangeLog]: fixed

**Comment by tianon** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**